EntAppProject/src/main/resources/templates/JBCreateAccount.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 
 <html>
-
 <head>
+<meta charset="ISO-8859-1"/>
+<link rel="stylesheet" href="css/style.css"/>

```

Comment:
This is a more traditional name and location for css... doing that will shorten the learning curve for new starters.